hack/install.sh

```diff
@@ -18,6 +18,17 @@ err() {
   exit 1
 }
 
+
+# Function to stop Daytona server if running
+stop_daytona_server() {
+  if pgrep -x "daytona" > /dev/null; then
+    echo "Daytona server is running, stopping it..."
+    pkill -x "daytona" || err "Failed to stop Daytona server"
```

Comment:
This will not solve the issue as the server will be restarted if it was started in daemon mode with `daytona server`.

We should handle the case where the user started the server with `daytona server` by executing `daytona server stop`.
If the user started the server with `daytona serve`, then `pkill` might be an okay solution.

Reply:
Will update my PR in sometime

Reply:
@ezhil56x this now only covers the case where `daytona server` was ran. As per my previous comment, we should handle both cases.

What I suggest is this:
1. Inform the user that we detected that the server is running.
2. Ask them if they want us to stop the server automatically. (yes/no prompt - this should be skippable via a `-y` flag)
3. If the user selects `no`, we abort the installation process and ask them to stop manually before trying again
4. If the user selects `yes`, we first attempt to stop the server with `daytona server stop`, if that fails, the server was probably started in the foreground with `daytona serve`. Then we should attempt with `pkill`.

Does this make sense?

Reply:
So it should handle both `daytona serve` and `daytona server`. Makes sense, will update my PR

Reply:
Updated!